xcodeproj/internal/bazel_integration_files/copy_outputs.sh

```diff
@@ -19,6 +19,21 @@ else
       suffix=/Payload
     fi
 
+    if [[ -n ${BAZEL_OUTPUTS_DSYM:-} ]]; then
+      while IFS='' read -r file; do dsym_files+=("${file}"); done < <(xargs -n1 <<< "$BAZEL_OUTPUTS_DSYM")
+      rsync \
+      --copy-links \
+      --recursive \
+      --times \
+      --archive \
+      --delete \
+      ${exclude_list:+--exclude-from="$exclude_list"} \
+      --chmod=u+w \
+      --out-format="%n%L" \
+      ${dsym_files[@]} \
+      "$TARGET_BUILD_DIR"
```

Comment:
Glad you found the other examples.

I don't think we need to create an array for this one though, I think instead you can do this inline:

```suggestion
      while IFS='' read -r file; do dsym_files+=("${file}"); done < <(xargs -n1 <<< "$BAZEL_OUTPUTS_DSYM")
      rsync \
      --copy-links \
      --recursive \
      --times \
      --archive \
      --delete \
      ${exclude_list:+--exclude-from="$exclude_list"} \
      --chmod=u+w \
      --out-format="%n%L" \
      $(xargs -n1 <<< "$BAZEL_OUTPUTS_DSYM") \
      "$TARGET_BUILD_DIR"
```

Which is how the preview frameworks used to be: https://github.com/buildbuddy-io/rules_xcodeproj/blob/b6144935ad30cb933d92f9c5579426f4868f93c2/xcodeproj/internal/bazel_integration_files/copy_outputs.sh#L72